actions/push-to-gar-docker/action.yaml

```diff
@@ -4,26 +4,63 @@ inputs:
   registry:
     description: |
       Google Artifact Registry to store docker images in.
+    default: "us-docker.pkg.dev"
   tags:
     description: |
       List of Docker images to be pushed.
-  build_path:
+    required: true
+  context:
     description: |
       Path to Dockerfile, which is used to build the image.
+    default: "."
   environment:
     description: |
       Environment for pushing artifacts (can be either dev or prod).
     default: dev
+  image_name:
+    description: |
+      Name of the image to be pushed to GAR.
+    required: true
 
 runs:
   using: composite
   steps:
+    - name: Resolve GCP project
+      id: resolve-project
+      shell: bash
+      run: |
+        if [ "${{ inputs.environment }}" == 'dev' ]; then
+          PROJECT="grafanalabs-dev"
+        elif [ "${{ inputs.environment }}" == 'prod' ]; then
+          PROJECT="grafanalabs-global"
+        else
+          echo "Invalid environment. Valid environment variable inputs: dev, prod"
+          exit 1
+        fi
+        echo "project=${PROJECT}" >> ${GITHUB_OUTPUT}
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+      with:
+        images: "${{ inputs.registry }}/${{ steps.resolve-project.outputs.project }}/${ROOT_REPO}-${{ inputs.environment }}/${{ inputs.image_name }}"
+        tags: ${{ inputs.tags }}
+    - name: Echo tags
```

Comment:
Do you still need this? IIRC docker/metadata-action already provides some useful output here. Or would it perhaps make sense here to use `$GITHUB_STEP_SUMMARY` for this?

Reply:
done!